.github/workflows/cml-report.yml

```diff
@@ -0,0 +1,100 @@
+name: CML Report
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  sim-summary:
+    if:  ${{ github.event.label.name == 'run-simulation' }}
+    name: CML Simulation
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    
+    - name: Build API
+      run: |
+        python local/rest_api_gcbm/cml_report/process_input.py
+        cd local/rest_api_gcbm
+        docker build -f Dockerfile.cml --build-arg BUILD_TYPE=RELEASE --build-arg NUM_CPU=4 -t gcbm-api .
+        docker run -d --name gcbm-api --rm -p 8080:8080 gcbm-api
+
+    - name: Get Public IP
+      id: ip
+      uses: haythem/public-ip@v1.2
+
+    - name: Print public IP
+      run: |
+        echo ${{ steps.ip.outputs.ipv4 }}
+    
+    - name: Check ports
+      run: |
+        sudo netstat -tunlp
+
+    - name: Create New Sim
+      run: |
+        curl -d "title=run4" -X POST http://0.0.0.0:8080/gcbm/new
+        
+    - name: Upload Files 
+      run: |
+        cd GCBM/GCBM_New_Demo_Run

```

Comment:
> There's a discussion going on at [moja-global/FLINT-UI#330](https://github.com/moja-global/FLINT-UI/discussions/330) about refactoring these endpoints.
> 
> One thing we should suggest is that the pre-processing logic be decoupled from the Flask endpoints. Then we run this action inside the `rest_api_gcbm` container and using a short python script to create the GCBM Demo Run configuration, rather than mocking up an HTTP request. This would allow us to run the simulation using `moja.cli` interactively, and solve the problem of having to probe whether the simulation is complete.
> 
> This can be tackled in a future PR.

So then we would only use the `rest_api_gcbm` for the upload of inputs and download of results?